language-design.md

```diff
@@ -74,6 +123,23 @@ the commenters?
 
 **Response**
 
+The comments on the Pavlus article generally seem to suggest that the article does not take many factors into consideration when claiming that programming languages should be more semantically intuitive. Pavlus insists that existing languages are very difficult to understand and have sacrificed usability for succinctness. He cites Perl for its confusing syntax and uses it as an example of how inaccessible programming languages are to the average person. 
+
+In order to address the conflict in the comments, we must look at the distinction between professional programmers and everyone else. Pavlus’ article says,
+
+> Unless you’ve had specialized training, looking at lines of code is like reading hieroglyphs, only less intuitive. [Pavlus, 2012]
+
+This defines professional programmers as those who have had specialized training and are thus able to understand code. Keeping this definition in mind, the majority of comments seemed to sympathize with professional programmers. While Pavlus mentions the need for a more intuitive programming language, the commenters cite the convenience of efficient syntax in writing professional programs. One commenter, Mjaksen, writes the following:
+
+> True but irrelevant. C/C++ *IS* a terrible language to express oneself in, but it's pretty good at 
+> organizing large software projects and interfacing with all the link libraries and shared libraries 
+> that it needs to get the job done… I have *never* seen a language that is concise, contextually 
+> clear, and easy to read, that can also organize code/objects/threads/errors and place nice with 
+> the os/shared libs. [Pavlus, 2012]
+
+The point made here is that it is difficult to have the best of both worlds. Programming languages are designed with efficiency in mind and make sacrifices in terms of readability.
+
```

Comment:
If the choice is between efficiency and readability, it makes sense to prioritize efficiency in more powerful languages like C/C++. However, I find it hard to believe that all of the unreadableness is purely due to designing for efficiency. It seems more like the designers planned for efficiency and forgot to even consider readability in the process.